components/about.tsx

```diff
@@ -0,0 +1,12 @@
+import Link from "next/link";
+
+export default function About() {
+  return (
+    <div>
+      <p className="text-2xl m-2">Information about the project coming soon</p>
```

Comment:
oh so you're down with tailwind?

Reply:
haha I just used it because you had already included it 😅 no real preference on my end